diff --git a/CMakeLists.txt b/CMakeLists.txt
index 20826ba3..f397684b 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -412,10 +412,8 @@ set(PROJ_DICTIONARY
 
 include_directories(${PROJ_SOURCE_DIR}/src)
 
-add_subdirectory(data)
 add_subdirectory(include)
 add_subdirectory(src)
-add_subdirectory(man)
 add_subdirectory(cmake)
 if(BUILD_TESTING)
   add_subdirectory(test)
@@ -428,10 +426,6 @@ endif()
 
 add_subdirectory(scripts)
 
-set(docfiles COPYING NEWS.md AUTHORS.md)
-install(FILES ${docfiles}
-        DESTINATION ${CMAKE_INSTALL_DOCDIR})
-
 ################################################################################
 # Build debug symbols
 ################################################################################
