diff --git a/main.mk b/main.mk
index a0b08e5773..1b7620e303 100644
--- a/main.mk
+++ b/main.mk
@@ -1994,9 +1994,9 @@ xbin: threadtest5
 # runtime performance hit, which is fine for use in the shell but is
 # not appropriate for the canonical library build.
 #
-sqlite3$(T.exe):	shell.c sqlite3.c
+sqlite3$(T.exe):	sqlite3.c
 	$(T.link) -o $@ \
-		shell.c sqlite3.c \
+		sqlite3.c \
 		$(CFLAGS.readline) $(SHELL_OPT) $(CFLAGS.icu) \
 		$(LDFLAGS.libsqlite3) $(LDFLAGS.readline)
 #
@@ -2352,7 +2352,7 @@ distclean:	distclean-. clean
 
 
 # Show important variable settings.
-show-variables:	
+show-variables:
 	@echo "CC          = $(CC)"
 	@echo "B.cc        = $(B.cc)"
 	@echo "T.cc        = $(T.cc)"
