diff --git a/Makefile.in b/Makefile.in
index 8ffc77750d..f4697e1056 100644
--- a/Makefile.in
+++ b/Makefile.in
@@ -560,9 +560,9 @@ libtclsqlite3.la:	tclsqlite.lo libsqlite3.la
 		-version-info "8:6:8" \
 		-avoid-version

-sqlite3$(TEXE):	shell.c sqlite3.c
+sqlite3$(TEXE):	sqlite3.c
 	$(LTLINK) $(READLINE_FLAGS) $(SHELL_OPT) -o $@ \
-		shell.c sqlite3.c \
+		sqlite3.c \
 		$(LIBREADLINE) $(TLIBS) -rpath "$(libdir)"

 sqldiff$(TEXE):	$(TOP)/tool/sqldiff.c sqlite3.lo sqlite3.h
